api/jobs-api/src/main/java/com/cognifide/aet/job/api/collector/JsErrorLog.java

```diff
@@ -74,4 +71,13 @@ public int compareTo(JsErrorLog o) {
     return new CompareToBuilder().append(sourceName, o.sourceName).append(lineNumber, o.lineNumber)
         .append(errorMessage, o.errorMessage).toComparison();
   }
+
+  @Override
+  public String toString() {
```

Comment:
This method is not used anywhere.
Why is it here? We don;t have `toString` methods in other classes in this package... :?
Maybe we don;t need it and could remove this change?

Reply:
deleted